deltachat-server/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "deltachat-server"
```

Comment:
this is bikeshedding, but i think this is a bad name.  even `deltachat-jsonrpc-server` would be better.

Reply:
sure, `deltachat-rpc-server` is also fine.  I just fear that `deltachat-server` suggests something entirely different, people might get confused and think deltachat now needs a server component in the worst-case of the different meanings it could have.